### PR TITLE
docs/variants/msi: Add heads documentation

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Comment by mkopec** [comment]
@BeataZdunczyk Do we want this merged now, or after release notes are added and tests are completed?

**Review by mkopec — APPROVED** [review]

File: docs/variants/msi_z690/heads.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Dasharo (coreboot + Heads) firmware variant
+
+Heads-based variant of Dasharo firmware compatible with MSI PRO Z690-A boards
+is offered as a Technology Preview Release.
+
+Please consider to support the project financially by purchasing the
+[Dasharo Entry Subscription](https://novacustom.com/product/dasharo-entry-subscription/).
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/variants/msi_z690/heads.md:7]: The base for the file was copy-pasted from https://github.com/Dasharo/docs/pull/754

My bad

[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/variants/msi_z690/heads.md]: Removed: 5cf4a18c8df8627e12a14e5b663f5038198c66a2

File: docs/variants/msi_z690/heads.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+The subscription is not automatically renewed.
+
+## Releases
+
+Following Release Notes describe status of Dasharo (coreboot + Heads) firmware
+development compatible with MSI PRO Z690-A boards.
+
+### v0.9.0
+
+TBD
```
Comment (miczyg1) [inline docs/variants/msi_z690/heads.md:20]:
```suggestion
COMING SOON
```

File: docs/variants/msi_z690/heads.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+* Docker
+    - follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    - follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* Git
+
+    ```bash
+    sudo apt -y install git
+    ```
+
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z690/heads.md:54]:
```suggestion
### Building 

```

Add a "Building" header before step one.

File: docs/variants/msi_z690/heads.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    `build/x86/msi_z690a_ddr5/dasharo-msi_z690a_ddr5-*.rom`.
+
+## Transition from Dasharo UEFI to Heads variant
+
+**Recommended**
+
+Please use [Dasharo Tools Suite](../../dasharo-tools-suite/overview.md) with
+Heads subscription credentials to transition to Heads.
+
+**Manual**
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z690/heads.md:102]: Let's recommend switching back to AMI firmware before using FlashBIOS with the heads binary and povide explanations or include links to instructions on how to perform these steps.

Consider incorporating admonitions from https://squidfunk.github.io/mkdocs-material/reference/admonitions/ to grab the user's attention, similar to what we did here: https://docs.dasharo.com/variants/novacustom_nv4x_adl/heads/#switching-from-dasharo-uefi-to-heads

File: docs/variants/msi_z690/heads.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+Generally, the same [hardware configuration](hardware-matrix.md) as for the
+UEFI variant applies.
+
+A notable addition is usage of the
+[Nitrokey 3A Mini](https://shop.nitrokey.com/shop/nk3am-nitrokey-3a-mini-149)
+USB device, which is required for Heads installation and usage.
+
+## Test Matrix
+
+Please refer to the [tests results spreadsheet](TBD).
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z690/heads.md:31]:
```suggestion
Please refer to the [tests results spreadsheet](https://docs.google.com/spreadsheets/d/1yWZ--zFPIsQhXZByf7nJIrasQYuRSf1yCi60lY_RGsQ/edit#gid=5649308).
```

File: docs/variants/msi_z790/heads.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+Generally, the same [hardware configuration](hardware-matrix.md) as for the
+UEFI variant applies.
+
+A notable addition is usage of the
+[Nitrokey 3A Mini](https://shop.nitrokey.com/shop/nk3am-nitrokey-3a-mini-149)
+USB device, which is required for Heads installation and usage.
+
+## Test Matrix
+
+Please refer to the [tests results spreadsheet](TBD).
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z790/heads.md:33]:
```suggestion
Please refer to the [tests results spreadsheet](https://docs.google.com/spreadsheets/d/1yWZ--zFPIsQhXZByf7nJIrasQYuRSf1yCi60lY_RGsQ/edit#gid=5649308).
```

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: docs/variants/msi_z690/heads.md (hunk 3 above)
Comment (miczyg1) [reply docs/variants/msi_z690/heads.md:108]: https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-325e58bb2b51acd669b828c9d7e5f51ebc864ab5da29674d065c031e7c420553R109

File: docs/variants/msi_z690/heads.md (hunk 4 above)
Comment (miczyg1) [reply docs/variants/msi_z690/heads.md:158]: https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-325e58bb2b51acd669b828c9d7e5f51ebc864ab5da29674d065c031e7c420553R109

https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-325e58bb2b51acd669b828c9d7e5f51ebc864ab5da29674d065c031e7c420553R172

Comment (miczyg1) [reply docs/variants/msi_z690/heads.md:158]: https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-66eaec24cfbda8a20d1ec829dfe08040814f5bd46b20274382f9092a3d784e7bR172-R177
https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-66eaec24cfbda8a20d1ec829dfe08040814f5bd46b20274382f9092a3d784e7bR161-R166

File: docs/variants/msi_z690/heads.md (hunk 5 above)
Comment (miczyg1) [reply docs/variants/msi_z690/heads.md]: https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-325e58bb2b51acd669b828c9d7e5f51ebc864ab5da29674d065c031e7c420553R85

File: docs/variants/msi_z790/heads.md (hunk 6 above)
Comment (miczyg1) [reply docs/variants/msi_z790/heads.md]: https://github.com/Dasharo/docs/pull/762/commits/7f98b8b727264eed371584e8b4ee364d2db34c2d#diff-66eaec24cfbda8a20d1ec829dfe08040814f5bd46b20274382f9092a3d784e7bR85

File: docs/variants/msi_z690/heads.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+### Building
+
+1. Clone Dasharo Heads repository
+
+   ```bash
+    git clone https://github.com/Dasharo/heads.git
+   ```
+
+2. Navigate to the source code directory and checkout to the desired revision:
```
Comment (BeataZdunczyk) [inline docs/variants/msi_z690/heads.md:117]: It renders as '1.' in my local preview for both platforms.

File: docs/variants/msi_z690/heads.md (hunk 4 above)
Comment (BeataZdunczyk) [reply docs/variants/msi_z690/heads.md:158]: great, resolving

**Review by BeataZdunczyk — CHANGES_REQUESTED** [review]

File: docs/variants/msi_z690/heads.md (hunk 7 above)
Comment (miczyg1) [reply docs/variants/msi_z690/heads.md:117]: Fixed [152a450](https://github.com/Dasharo/docs/pull/762/commits/152a450d41fde23922d75d58e3d57fbc6ab9c1f2)

**Comment by miczyg1** [comment]
@BeataZdunczyk release notes updated to the most recent version

**Review by BeataZdunczyk — APPROVED** [review]

**Review by mkopec — APPROVED** [review]

**Comment by BeataZdunczyk** [comment]
LGTM

**Pull request merged**